resources/codebuild-source-credentials.go

```diff
@@ -0,0 +1,54 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/codebuild"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type CodeBuildSourceCredential struct {
+	svc        *codebuild.CodeBuild
+	Arn        *string
+	AuthType   *string
+	ServerType *string
+}
+
+func init() {
+	register("CodeBuildSourceCredential", ListCodeBuildSourceCredential)
+}
+
+func ListCodeBuildSourceCredential(sess *session.Session) ([]Resource, error) {
+	svc := codebuild.New(sess)
+	resources := []Resource{}
+
+	params := &codebuild.ListSourceCredentialsInput{}
+
+	resp, err := svc.ListSourceCredentials(params)
```

Comment:
Is there no pagination here required?

Reply:
@bjoernhaeuser this endpoint is not paginated, [SourceCredentialsInfo](https://docs.aws.amazon.com/sdk-for-go/api/service/codebuild/#SourceCredentialsInfo) doesn't have a `NextToken` field.